_sass/base/_markdown.scss

```diff
@@ -26,13 +26,15 @@
   // List Items
   li {
     line-height: 24px;
+    margin-top:12px;
+
   }
 
   // Lists
   ol,
   ul {
     list-style: initial;
-    list-style-position: inside;
+    list-style-position: outside;
```

Comment:
This seems unrelated.

Reply:
This is actually a related fix, to solve tkoeppe's issue with bullets appearing on a separate line from the bullet text (which would otherwise occur again in this PR).